src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -2153,10 +2159,16 @@ private unsafe ParsingError PrivateParseMinimal()
                 {
                     return ParsingError.BadAuthority;
                 }
+                // There is no Authority component, save the Path index
+                // mailto is treated specially.
                 else if (_syntax.NotAny(UriSyntaxFlags.MailToLikeUri))
                 {
-                    // There is no Authority component, save the Path index
-                    // mailto is treated specially.
+                    // By now we know the URI has no Authority, so if the URI must be normalized, initialize it without one.
+                    if (_iriParsing && (_flags & Flags.HasUnicode) != 0 && (_flags & Flags.HostUnicodeNormalized) == 0)
+                    {
+                        _string = _string.Substring(0, idx);
```

Comment:
Same question.

Reply:
Same situation as above, but occurs when the scheme is followed by "///" to specify an empty host.